lib/src/model/tournament/tournament.dart

```diff
@@ -355,11 +356,12 @@ FeaturedGame _featuredGameFromPick(RequiredPick pick) {
     orientation: pick('orientation').asSideOrThrow(),
     fen: pick('fen').asStringOrThrow(),
     lastMove: pick('lastMove').asUciMoveOrNull(),
-    finished: pick('finished').asBoolOrNull(),
```

Comment:
Might still want to keep it. Perhaps I'm wrong but if server provides that fields doesn't it mean that absence of clocks is not enough to determine a tournament is finished on some cases?

Reply:
I'm not sure why I thought so, but the server does not actually provide this field: https://github.com/lichess-org/lila/blob/cc2156805a95b5156e096166db067fd7c8950a07/ui/tournament/src/interfaces.ts#L100

As far as I understand it, you can fully determine the state from the combination of `clocks`, `winner` and `fen`

1) If `clocks` is not null, the game is either started or not started yet
   a) If the FEN is the initial FEN, the game is not started (yes, this breaks for some variants, but lila uses the same heuristic and most tournament games start quickly)
   b) Otherwise, the game is started.
2) If `clocks` is null and `winner` is null, the game is finished and it's a draw
3) Otherwise, the game is finished and `winner` tells us who won

Reply:
Oh I thought it was for the tournament, but it is for the featured game. Fine then.